compiler/main/driver.cpp

```diff
@@ -619,8 +619,8 @@ static void runCompilerInLLDB(int argc, char* argv[]) {
 
 static void readConfig(const ArgumentDescription* desc, const char* arg_unused) {
   // Expect arg_unused to be a string of either of these forms:
-  // 1. name=value -- set the config param "name" to "value"
-  // 2. name       -- set the boolean config param "name" to NOT("name")
+  // 1. name=value -- set the config "name" to "value"
+  // 2. name       -- set the boolean config "name" to 'true'
   //                  if name is not type bool, set it to 0.
```

Comment:
The second half of comment 2 is also no longer true.  It sets non-bools to `true` too.